contrib/helix/template.py

```diff
@@ -0,0 +1,37 @@
+pkgname = "helix"
+pkgver = "23.05"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo", "git"]
+makedepends = ["rust"]
+pkgdesc = "Fast modal terminal-based text editor"
+maintainer = "Wesley Moore <wes@wezm.net>"
+license = "MPL-2.0"
+url = "https://github.com/helix-editor/helix"
+source = f"{url}/releases/download/{pkgver}/helix-{pkgver}-source.tar.xz"
+sha256 = "c1ca69facde99d708175c686ce5bf3585e119e372c83e1c3dc1d562c7a8e3d87"
+
+
+def do_install(self):
+    self.cargo.install(wrksrc="helix-term")
```

Comment:
If alice's change in #259 is merged this can be tweaked to use `cargo_build_path`.

Reply:
didn't know this was even possible, i guess this also technically works for the original issue. not sure which is better :)

Reply:
let's just get this in as is for now